packages/components/src/number-control/index.js

```diff
@@ -101,7 +105,9 @@ export function NumberControl(
 		if ( type === inputControlActionTypes.DRAG && isDragEnabled ) {
 			const { delta, shiftKey } = payload;
 			const [ x, y ] = delta;
-			const modifier = shiftKey ? shiftStep : 1;
+			const modifier = shiftKey
```

Comment:
Improvement: The modifier now accounts for shiftStep **and** step (used as a multiplier)